build.psm1

```diff
@@ -1188,6 +1188,53 @@ function Show-PSPesterError
 
 }
 
+function Test-XUnitTestResults
+{
+    param(
+        [Parameter(Mandatory)]
+        [ValidateNotNullOrEmpty()]
+        [string] $TestResultsFile
+    )
+
+    if(-not (Test-Path $TestResultsFile))
+    {
+        throw "File not found $TestResultsFile"
+    }
+
+    try
+    {
+        $results = [xml] (Get-Content $TestResultsFile)
+    }
+    catch
+    {
+        throw "Cannot convert $TestResultsFile to xml : $($_.message)"
+    }
+
+    $failedTests = $results.'assemblies'.'assembly'.'collection' | Where-Object failed -gt 0
```

Comment:
maybe the single quote is not needed ...

Reply:
Fixed.